desc/objectives/_coils.py

```diff
@@ -238,6 +239,7 @@ def compute(self, params, constants=None):
             params=params,
             transforms=constants["transforms"],
             grid=self._grid,
+            basis=kwargs.get("basis", None),
```

Comment:
why default to `None` here?

Reply:
I actually made a mistake here, if None gets passed in for basis, we get an error because the function is not equipped to handle it and does not have a default. I might just default to xyz here but not sure yet of which is best.